ext_mongodb.php

```diff
@@ -1,6 +1,12 @@
 <?hh
 namespace MongoDB\Driver;
 
+interface TypeWrapper
+{
+	function fromType(\MongoDB\BSON\Type $type);
+	function toType() : \MongoDB\BSON\Type;
```

Comment:
I've made an effort of using FQCNs everywhere. More out of readability and knowing wtf goes on :-)